packages/create-catalyst/src/commands/create.ts

```diff
@@ -24,6 +24,7 @@ export const create = new Command('create')
   .option('--channel-id <id>', 'BigCommerce channel ID')
   .option('--customer-impersonation-token <token>', 'BigCommerce customer impersonation token')
   .option('--gh-ref <ref>', 'Clone a specific ref from the bigcommerce/catalyst repository')
```

Comment:
Should I adjust this description since now any repository can be cloned?

Reply:
Yeah, I'd adjust it.